tutor/src/components/course-calendar/index.jsx

```diff
@@ -38,11 +38,6 @@ const getDisplayBounds = {
   },
 };
 
-function addPlanToListing(plan) {
-  TaskPlans
-    .forCourseId(this.props.params.courseId)
-    .addPublishingPlan(plan);
-}
```

Comment:
I removed this because it's never called and has a bug.  `this.props.params.courseId` will be invalid, since `addPlanToListing` is a standalone function.